include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -7257,9 +7257,14 @@
       "class"  : "Type-Declaration",
       "opcode" : 6086,
       "operands" : [
-        { "kind" : "IdResult" }
+        { "kind" : "IdResult" },
+        {
+          "kind" : "AccessQualifier",
+          "name" : "'AccessQualifier'"
+        }
       ],
       "capabilities" : [ "VectorComputeINTEL" ],
+      "extensions" : [ "SPV_INTEL_vector_compute" ],
```

Comment:
It's advised not to mention the extension during instruction declaration, instead it's kept in capability-extension map.

Reply:
Fixed